src/components/JobCard/JobCard.tsx

```diff
@@ -1,68 +1,77 @@
 import { Text, View, Pressable, Modal } from 'react-native';
 import styles from './CardStyles';
 import React, { useState } from 'react';
+import GestureRecognizer from 'react-native-swipe-gestures';
 
 interface JobCardProps {
-  title: string;
   description: string;
-  employer: string;
   hours: number;
-  salary: number;
+  employer: string;
   contact_info: string;
-  end_date: Date;
-  job_creator: string;
-  start_date: string;
+  id: string;
+  date: string;
+  companyName: string;
+  address: string;
+  contactPerson: string;
+  phone: string;
+  jobPosition: string;
+  languageRequirement: string;
+  workingHours: string;
+  workingDays: string;
+  salary: string;
+  probationPeriod: string;
+  employeeBenefit: string;
+  otherInfo: string;
+  hidden: Map<string, boolean>;
 }
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type, @typescript-eslint/naming-convention
-const JobCard = ({ title, description, employer, hours, salary, contact_info }: JobCardProps) => {
+const JobCard = ({ title, description, employer, hours, salary, contact_info, hidden }: JobCardProps) => {
```

Comment:
just wanna check: do you have prettier installed? this is automatically reformatted for me when I save this file (each field is on a new line)